datafusion/common/src/config.rs

```diff
@@ -453,11 +453,13 @@ config_namespace! {
         /// ```
         pub repartition_sorts: bool, default = true
 
-        /// When true, DataFusion will opportunistically remove sorts by replacing
-        /// `RepartitionExec` with `SortPreservingRepartitionExec`, and
+        /// When true, DataFusion will opportunistically remove sorts when the data is already sorted,
+        /// replacing `RepartitionExec` with `SortPreservingRepartitionExec`, and
```

Comment:
Hmm, actually there is no `SortPreservingRepartitionExec` operator but it is a variant of `RepartitionExec` with `preserve_order` as true. It is a little confusion at first if trying to look for `SortPreservingRepartitionExec` type in IDE.

Maybe:

```suggestion
        /// replacing `RepartitionExec` with `SortPreservingRepartitionExec` (i.e., `RepartitionExec` with `preserve_order` as true), and
```

Reply:
That makes sense, thank you @viirya  --  I tried to improve the wording in 35c674852b. Let me know what you think

Reply:
Looks good to me. Thanks @alamb